index.html

```diff
@@ -46,17 +46,15 @@
   </head>
   <body>
     <div class="card">
-      <img src="my-profile-picture.png" alt="Girlcoder" style="width: 100%" />
-      <h1>Zandile Doe</h1>
+      <img src="/my-profile-picture.jpg" alt="Mita" style="width: 100%" />
+      <h1>Mita Raboya</h1>
       <p class="title">Software developer</p>
-      <p>Girlcode Bootcamp</p>
+      <p>Girlcode Learnership</p>
       <div style="margin: 24px 0">
-        <a href="https://twitter.com/GirlCode_za"><i class="fa fa-twitter"></i></a>
-        <a href="https://www.linkedin.com/company/girlcodeza"><i class="fa fa-linkedin"></i></a>
-        <a href="https://github.com/GirlCodeZA"><i class="fa fa-github"></i></a>
+        <a href="https://twitter.com/Mita_Raboya"><i class="fa fa-twitter"></i></a>
       </div>
       <p>
-        <a href="mailto:girlcode@gmail.com"><button>Contact</button></a>
+        <a href="mailto:mitaraboya@gmail.com@gmail.com"><button>Contact</button></a>
```

Comment:
This file should not be changed, please revert.

Your changes should only contain:
- your profile pic in 2021/images
- the HTML block in the `yearbook.html` that you correctly did 

I would advise you start from scratch after saving your jpg file and your HTML code to add in order to make a clean pull request with the only required changes.

Reply:
You are also there !!